lib/oli_web/live/delivery/instructor_dashboard/helpers.ex

```diff
@@ -60,10 +60,8 @@ defmodule OliWeb.Delivery.InstructorDashboard.Helpers do
   end
 
   def get_assessments(section, students) do
-    graded_pages_and_section_resources =
-      DeliveryResolver.graded_pages_revisions_and_section_resources(section.slug)
```

Comment:
The first query that is run - a "expensive" resolver query to fetch Revisions + SectionResource records.